src/borg/remote.py

```diff
@@ -901,7 +901,7 @@ def check(self, repair=False, save_space=False, max_duration=0):
     @api(since=parse_version('1.0.0'),
          compact={'since': parse_version('1.2.0a0'), 'previously': True, 'dontcare': True},
          cleanup_commits={'since': parse_version('1.2.0a0'), 'previously': False, 'dontcare': True})
-    def commit(self, save_space=False, compact=True, cleanup_commits=False):
+    def commit(self, save_space=False, compact=True, threshold=0.1, cleanup_commits=False):
```

Comment:
you need to extend the @api decoration in line 901+ also.

check the decorator code for the meaning of the misc. values.